ipatool

```diff
@@ -877,6 +877,28 @@ def pr_push_command(ctx):
     if 'pushed' in labels:
         ctx.die('Pull request was already pushed')
 
+    if not pr.mergeable:
+        ctx.die('Pull request is not mergeable.')
+
+    # TODO: use with CombinedStatus once released
+    # status = repo.commit(pr.head.sha).status()
+    # if status.state in ('error', 'failure',):
+    #     ctx.die('Pull request failed CI test(s)')
+    # elif status.state == 'pending':
+    #     ctx.die('CI have not completed testing the pull request yet')
+    # remove vvv
+    most_recent_results = {}
+    for s in repo.commit(pr.head.sha).statuses():
```

Comment:
From official documentation [1]:
Statuses are returned in reverse chronological order. The first status in the list will be the latest one.

[1] https://developer.github.com/v3/repos/statuses/#list-statuses-for-a-specific-ref